pkgs/by-name/zt/ztools/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  unixtools,
+  groff,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ztools";
+  version = "7/3.1";
+
+  src = fetchurl {
+    url = "http://mirror.ifarchive.org/if-archive/infocom/tools/ztools/ztools731.tar.gz";
+    hash = "sha256-vlQX0/fCAr88KJwMnYUSROFOg9tfVK5Hz58AUDuhNXg=";
+  };
+
+  sourceRoot = ".";
+
+  nativeBuildInputs = [
+    groff
+    unixtools.col
+  ];
+
+  # compiler flags as defaults have changed
+  NIX_CFLAGS_COMPILE = "-Wno-implicit-function-declaration -Wno-implicit-int";
+
+  installPhase = ''
+    mkdir -p $out/bin/
+    cp {check,infodump,pix2gif,txd} $out/bin/
+  '';
```

Gotta run the pre and post flags.

```suggestion
  installPhase = ''
    runHook preInstall
    mkdir -p $out/bin/
    cp {check,infodump,pix2gif,txd} $out/bin/
    runHook postInstall
  '';